### PR TITLE
[Android][iOS] Clarify IAM policy update step

### LINKED ISSUE #675: Where to apply the IAM Policy referenced in Analytics setup docs?
**Is your feature request related to a problem? Please describe.**
The document here: https://aws-amplify.github.io/docs/ios/analytics#update-your-iam-policy lets us know, "Amazon Pinpoint service requires an IAM policy in order to use the submitEvents API," but it doesn't give clarity on where to apply this. The heading says something about updating your IAM Policy, but which one?

**Describe the solution you'd like**
It would be nice to understand a little more about this new policy, for clarity and confidence sake. Do I simply create the new policy and nothing further? Does this not need to be assigned to another IAM element? Should I select Pinpoint as the service when creating this? Is this simply for later use via reference?

### PR DESCRIPTION
**Pull request by kvasukib**
Fixes #675

*Issue #, if available:*

#675 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: android/analytics.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ The Amplify CLI helps setup and configure Pinpoint within your application and c
 
 #### Update your IAM Policy:
 
-Amazon Pinpoint service requires an IAM policy in order to use the `submitEvents` API:
+Amazon Pinpoint service requires an IAM policy in order to use the `submitEvents` API. If you are using long-term AWS credentials of an `Amazon IAM` user, attach the following policies to the role of the `IAM` user. If you are using temporary AWS credentials vended by `Amazon Cognito Identity Pools`, attach the following policies to the Unauthenticated and/or Authenticated `IAM` roles of your `Cognito Identity Pool`.
```
Comment (mlabieniec) [inline android/analytics.md]: The Amazon Pinpoint service requires permissions defined in an IAM policy to use the `submitEvents` API. If you are using long-term AWS credentials attached to an `Amazon IAM` user, attach the following policies to the role of that `IAM` user. If you are using temporary AWS credentials vended by `Amazon Cognito Identity Pools`, then attach the following policies to the Unauthenticated and/or Authenticated `IAM` roles of your `Cognito Identity Pool`. The role you attach the policies to depends on the scope of your application. For example, if you only want events submitted when users login, attach to the authenticated role. Similarly, if you want events submitted regardless of authentication state, attach the policy to the unauthenticated role. For more information on Cognito Identities authenticated/unauthenticated roles see <a href="https://docs.aws.amazon.com/cognito/latest/developerguide/identity-pools.html" target="_blank">here</a>.

File: ios/analytics.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ The Amplify CLI helps setup and configure Pinpoint within your application and c
 
 #### Update your IAM Policy:
 
-Amazon Pinpoint service requires an IAM policy in order to use the `submitEvents` API:
+Amazon Pinpoint service requires an IAM policy in order to use the `submitEvents` API. If you are using long-term AWS credentials of an `Amazon IAM` user, attach the following policies to the role of the `IAM` user. If you are using temporary AWS credentials vended by `Amazon Cognito Identity Pools`, attach the following policies to the Unauthenticated and/or Authenticated `IAM` roles of your `Cognito Identity Pool`.
```
Comment (mlabieniec) [inline ios/analytics.md]: Is this a duplicate of the above text? If so just use the same above

File: android/analytics.md (hunk 1 above)
Comment (kvasukib) [reply android/analytics.md]: Changed

File: ios/analytics.md (hunk 2 above)
Comment (kvasukib) [reply ios/analytics.md]: Yes it is. Updated the same.

**Pull request merged**